utils/src/ooinstall/oo_config.py

```diff
@@ -236,9 +241,36 @@ def _upgrade_legacy_config(self):
         self.settings['variant'] = 'openshift-enterprise'
         self.settings['variant_version'] = '3.0'
```

Comment:
Would this default version from the legacy config even work, or would it just error out?